uppsrc/CtrlCore/X11Ctrl.h

```diff
@@ -11,8 +11,8 @@
 
 private:
 	static ArrayMap<Window, XWindow>& Xwindow();
-	static int       WndCaretTime;
-	static bool      WndCaretVisible;
+//	static int       WndCaretTime;
+//	static bool      WndCaretVisible;
```

Comment:
Please remove commented code. We do not need it and there is no change that after applying this PR it will be needed in the future.

Reply:
Done.